doc/fuzzing.md

```diff
@@ -81,6 +81,17 @@ INFO: seed corpus: files: 991 min: 1b max: 1858b total: 288291b rss: 150Mb
 …
 ```
 
+## Reproduce a fuzzer crash found by the CI
```

Comment:
After a private discussion. "found" might confuse some readers, as the CI currently does not *search* for new inputs. It is repeating the existing test inputs like the unit tests.

Reply:
I hesitated on "found" as well..."reported"?